HousingFinanceInterimApi/serverless.yml

```diff
@@ -41,7 +41,8 @@ functions:
             resultTtlInSeconds: 0
             identitySource: method.request.header.Authorization
             managedExternally: true
-          cors: true
+          cors:
+            origin: '*'
```

Comment:
Sorry I meant the entire CORS section:
`            origin: '*'
            headers:
              - Content-Type
              - If-Match
              - X-Amz-Date
              - Authorization
              - X-Api-Key
              - X-Amz-Security-Token
              - X-Amz-User-Agent
            allowCredentials: false`